### ISSUE TITLE
Home Assistant 0.106 breaks battery card

### ISSUE THREAD

**Issue by stalakerob** [cbulock/lovelace-battery-entity#14]
Battery card not working under HASS 0.106

**Comment by teddythanh** [comment]
Yes. I have the same problem after upgraded to Hass 0.106. The card disappeared. Hic.

**Comment by jhbcosta** [comment]
Same problem here. After upgraded to 0.106 the card disappeared.

**Comment by Andrey1One** [comment]
Yes, not working under HA 0.106

**Comment by ostap414** [comment]
Есть такая проблема

**Comment by mattlward** [comment]
Same here, but shares a view with another card and it took it out as well.  The entire view is now blank.

**Comment by borland502** [comment]
The problem was fixed by the mini-chart developer which seemed to suffer the same issue: https://github.com/kalkih/mini-graph-card/tree/0.106.x-compatibility

**Comment by jarnose** [comment]
Change line 64 
from
`        this._config = config;`
to
`        this._config = JSON.parse(JSON.stringify(config));`

**Comment by liri** [comment]
created pull request:
https://github.com/cbulock/lovelace-battery-entity/pull/15

**Comment by ostap414** [comment]
> Change line 64
> from
> ` this._config = config;`
> to
> ` this._config = JSON.parse(JSON.stringify(config));`

It didn't help.The card doesn't work.

**Comment by Andrey1One** [comment]
Change line 64
This don't help me too

**Comment by liri** [comment]
It did solve the problem on my instance..

**Comment by teddythanh** [comment]
> Change line 64
> This don't help me too

It didn't work for me also.

**Comment by liri** [comment]
Did you try cleaning your cache or change the js file version in the url you have inside lovelace.yaml?

**Comment by ostap414** [comment]
The cache is cleaned. The version is 0.1.Not change.

**Comment by ostap414** [comment]
![01](https://user-images.githubusercontent.com/59996792/75649561-17a0ab00-5c86-11ea-9f0b-6f650e1ce6f1.png)
The Configurator doesn't like this line 110

**Comment by runnerfanspb** [comment]
Yes, not working under HA 0.106

**Comment by thehijacker** [comment]
> Change line 64
> from
> ` this._config = config;`
> to
> ` this._config = JSON.parse(JSON.stringify(config));`

This fixed it for me.

**Comment by ostap414** [comment]
Re-recorded again.Everything worked.

**Comment by teddythanh** [comment]
> Re-recorded again.Everything worked.

Sorry but what you mean "re-recorded"? :) How you fixed it? Thank you :)

**Comment by ostap414** [comment]
Deleted and created the card again

> > Re-recorded again.Everything worked.
> 
> Sorry but what you mean "re-recorded"? :) How you fixed it? Thank you :)

Deleted and created the card again

**Comment by teddythanh** [comment]
Finally I fixed it. Beside changing the code of line 64, we need to move the file battery-entity.js into www/community folder. Then change the yaml code in your ui-lovelace.yaml as below and it will work again.
```
resources:
   - url: /community_plugin/battery-entity.js?v=0.2
     type: module
```

**Comment by ostap414** [comment]
> Finally I fixed it. Beside changing the code of line 64, we need to move the file battery-entity.js into www/community folder. Then change the yaml code in your ui-lovelace.yaml as below and it will work again.
> 
> ```
> resources:
>    - url: /community_plugin/battery-entity.js?v=0.2
>      type: module
> ```

After updating 106.5 ,the card does not work

**Comment by chilicheech** [comment]
I just installed it using HACS and I'm getting this error:
```
Custom element doesn't exist: battery-entity.
type: 'custom:battery-entity'
entity: sensor.side_door_battery_level
```

**Comment by RamonM77** [comment]
The card seems to break the complete page when multiple entities are included. A single entity however doesn't break the page, but does show a part of the problem. The border is missing when a single entity is shown.
Hope this helps you to find a solution for this issue.

Example:
![battery-card](https://user-images.githubusercontent.com/54005671/76315243-9a55e580-62d8-11ea-9031-f10c0ab7d0df.png)

**Comment by mattlward** [comment]
Agreed, I am unable to add a secondary sensor as well.  Also, install does not place the correct entry in the lovelace config.

**Comment by richard-scott** [comment]
I've also tried this on Home Assistant 0.106.6, and the resource didn't get added to lovelace when clicking this:

![2020-03-12 14_11_07-Home Assistant](https://user-images.githubusercontent.com/5149834/76530401-95756b00-646b-11ea-9c72-683b57b222c3.png)

I have had to manually add this:

```
  - type: module
    url: /hacsfiles/lovelace-battery-entity/battery-entity.js
```

and once I did, it all worked!
